src/js/gameBuilder.js

```diff
@@ -44,144 +40,48 @@ export function buildGameTable(level) {
-            neighbor_2: "",
-            neighbor_3: "",
-            neighbor_4: {row: i, col: n - 1},
-            neighbor_5: {row: i, col: n + 1},        
-            neighbor_6: {row: i + 1, col: n - 1},
-            neighbor_7: {row: i + 1, col: n},
-            neighbor_8: {row: i + 1, col: n + 1}
-            }
-    } else if (i == 0 && n > 0 && i < lastNeighborY && n == lastNeighborX){
-        return {
-            neighbor_1: "",
-            neighbor_2: "",
-            neighbor_3: "",
-            neighbor_4: {row: i, col: n - 1},
-            neighbor_5: "",        
-            neighbor_6: {row: i + 1, col: n - 1},
-            neighbor_7: {row: i + 1, col: n},
-            neighbor_8: ""
-            } 
-    } else if (i > 0 && n == 0 && i < lastNeighborY && n < lastNeighborX) {
-        return {
-            neighbor_1: "",
-            neighbor_2: {row: i - 1, col: n},
-            neighbor_3: {row: i - 1, col: n + 1},
-            neighbor_4: "",
-            neighbor_5: {row: i, col: n + 1},        
-            neighbor_6: "",
-            neighbor_7: {row: i + 1, col: n},
-            neighbor_8: {row: i + 1, col: n + 1}
-            }
-    } else if (i > 0 && n == 0 && i == lastNeighborY && n < lastNeighborX) {
-        return {
-            neighbor_1: "",
-            neighbor_2: {row: i - 1, col: n},
-            neighbor_3: {row: i - 1, col: n + 1},
-            neighbor_4: "",
-            neighbor_5: {row: i, col: n + 1},        
-            neighbor_6: "",
-            neighbor_7: "",
-            neighbor_8: ""
-            }
-    } else if (i > 0 && n > 0 && i < lastNeighborY && n == lastNeighborX) {
-        return {
-            neighbor_1: {row: i - 1, col: n - 1},
-            neighbor_2: {row: i - 1, col: n},
-            neighbor_3: "",
-            neighbor_4: {row: i, col: n - 1},
-            neighbor_5: "",        
-            neighbor_6: {row: i + 1, col: n - 1},
-            neighbor_7: {row: i + 1, col: n},
-            neighbor_8: ""
-            }
-    } else if (i > 0 && n > 0 && i == lastNeighborY && n < lastNeighborX) {
-        return {
-            neighbor_1: {row: i - 1, col: n - 1},
-            neighbor_2: {row: i - 1, col: n},
-            neighbor_3: {row: i - 1, col: n + 1},
-            neighbor_4: {row: i, col: n - 1},
-            neighbor_5: {row: i, col: n + 1},        
-            neighbor_6: "",
-            neighbor_7: "",
-            neighbor_8: ""
-            }
-    } else if (i > 0 && n > 0 && i == lastNeighborY && n == lastNeighborX) {
-        return {
-            neighbor_1: {row: i - 1, col: n - 1},
-            neighbor_2: {row: i - 1, col: n},
-            neighbor_3: "",
-            neighbor_4: {row: i, col: n - 1},
-            neighbor_5: "",        
-            neighbor_6: "",
-            neighbor_7: "",
-            neighbor_8: ""
-            }
+    const out = Object.create(null);
+    let num = 1;
+
+    for (let x = i - 1; x <= i + 1; x++) {
```

Comment:
Well, the outcome of the following is exactly the same as the red lines above, just a bit more programmatically.